deployments/ansible/roles/collector/tasks/otel_win_install.yml

```diff
@@ -31,6 +31,29 @@
     state: present
   notify: "restart windows splunk-otel-collector"
 
+- name: Read the default config
+  slurp:
+    path: "{{splunk_otel_collector_config}}"
+  register: defaultconfig
+  when: splunk_config_override != ''
+
+- name: Extract the default config
+  set_fact:
+    updated_config: "{{ defaultconfig['content'] | b64decode | from_yaml }}"
+  when: splunk_config_override != ''
+
+- name: Update the default config
+  set_fact:
+    updated_config: "{{ updated_config | combine(splunk_config_override, recursive=True, list_merge='append_rp') }}"
```

Comment:
list_merge argument should be configurable with `splunk_config_override_list_merge` variable which should be `replace` by default not `append_rp`

Reply:
Using `list_merge=append_rp` we can able to add the receivers in to the `pipelines` section using the same `splunk_config_override` variable with out using another variable `splunk_config_override_list_merge`.

Reply:
How user can disable (exclude from the list) a component with `append_rp`?

Reply:
Ok,  Got it!
thanks.